app/modules/MenuCustom/MenuCustom.tsx

```
@@ -412,406 +257,40 @@ const MenuCustom = () => {
-                    </span>
-                  </div>
-                </div>
-              </CardContent>
-            </Card>
+            <MenuCard
+              item={item}
+              setEditingItem={setEditingItem}
+              setShowEditItemDialog={setShowEditItemDialog}
+              handleItemClick={handleItemClick}
+              handleDeleteMenuItem={handleDeleteMenuItem}
+            />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)